_includes/modern-appendix.md

```diff
@@ -221,6 +221,8 @@ We do try to note numerics where this is the case with a message like *"The text
 
 The first message sent after client registration, this message introduces the client to the network. The text used in the last param of this message varies wildly.
 
+The first parameter of this message is the nickname assigned by the network to the client. Since it may differ from the nickname the client requested with the `NICK` command (due to, e.g. server policy restrictions on nicknames), the client SHOULD use this parameter to determine its effective nickname at the time of connection. (Subsequent nickname changes, client-initiated or not, will be communicated by the server using a [NICK message](#nick-message).)
```

Comment:
```suggestion
The first parameter of this message is the nickname assigned by the network to the client. Since it may differ from the nickname the client requested with the `NICK` command (due to server policy, restrictions on nicknames, truncation) the client SHOULD use this parameter to determine its actual nickname at the time of connection. Subsequent nickname changes, client-initiated or not, will be communicated by the server sending a {% message NICK %} message.
```